### PR TITLE
changed the scorecard badge link to the standard format

### PR DESCRIPTION
**Pull request by harshitasao**
**Description of the changes being introduced by the pull request**:

Changes the scorecard badge link from old format to the Standard human-readable OpenSSF Scorecard Report.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Docs](https://readthedocs.org/projects/theupdateframework/badge/)](https://theupdateframework.readthedocs.io/)
 [![CII](https://bestpractices.coreinfrastructure.org/projects/1351/badge)](https://bestpractices.coreinfrastructure.org/projects/1351)
 [![PyPI](https://img.shields.io/pypi/v/tuf)](https://pypi.org/project/tuf/)
-[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/theupdateframework/python-tuf/badge)](https://api.securityscorecards.dev/projects/github.com/theupdateframework/python-tuf)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/theupdateframework/python-tuf/badge)](https://scorecard.dev/viewer/?uri=github.com/theupdateframework/python-tuf)
```
Comment (joshuagl) [inline README.md]: It looks like the scorecards domain changed. It used to be securityscorecards.dev (and that's still linked to from openssf.org), but the [project's README](https://github.com/ossf/scorecard?tab=readme-ov-file#view-a-projects-score) points to scorecard.dev

**Review by joshuagl — APPROVED** [review]
Thanks for the contribution!

**Comment by jku** [comment]
yeah, that looks a lot better, thanks. 

We should update the badge url as well though to be consistent. I can add that change since your branch allows it, just a sec...

**Comment by jku** [comment]
I don't understand why the CI is not running. I'll try closing and reopening to trigger a run

**Comment by jku** [comment]
Thanks!

**Pull request merged**